.gitignore

```diff
@@ -0,0 +1,5 @@
+.secrets
+bouncer/db.sqlite3
+bouncer/bouncer/__pycache__
```

Comment:
We don't want to include every directory separately, just adding `__pycache__` should do the trick.